### DISCUSSION TITLE
my statusline is strange

### DISCUSSION THREAD

**Discussion by zyxisme** [NvChad/NvChad#2645] [Q&A] [answered]
**WHY?**
![截图 2024-02-01 10-01-18](https://github.com/NvChad/NvChad/assets/103358586/593f80b9-0a61-4f93-b539-078d3ef9f210)

**Comment by siduck** [comment, marked as answer]
terminal issue

**Reply by zyxisme** [reply]
how to fix it?

**Reply by siduck** [reply]
@d116u set proper nerdfonts in your terminal, if you are on linux then use my st terminal build to get the best out of nvchad and my support :sunglasses:

**Reply by zyxisme** [reply]
thanks,and how to change statusline styles?
![www alltoall net_statuslines_a7IekrER4Z](https://github.com/NvChad/NvChad/assets/103358586/39b8e332-e635-4857-ac9c-c0c34be2ae8f)

**Reply by siduck** [reply]
@d116u read core/default_config.lua, its your friend along with nvchad.com

**Reply by zyxisme** [reply]
thanks.
![image](https://github.com/NvChad/NvChad/assets/103358586/c9edd1cb-fb33-4563-b1de-d230d077c5a5)
